METRICS.md

```diff
@@ -0,0 +1,101 @@
+_Last updated: May 20 2020_
+
+# Firefox Private Relay Analytics Plan
+
+This is the Analytics plan for Firefox Private Relay. It documents our use of Google Analytics and what we do with the information we collect.
+
+## Analysis
+
+**Private Relay uses Google Analytics to collect and organize data. We do this to get a better understanding of what is working, and where we still have work to do.**
+
+**Captured data also helps provide answers to the following questions:**
+
+**Demographic:**
+
+>From which country does the majority of our traffic originate?
+
+>Which browsers are most commonly used to access the Firefox Private Relay website?
+
+>Which devices are most commonly used to access the Firefox Private Relay website?
+
+**User Behavior:**
+
+>Do users delete aliases?
+
+>Do users create aliases?
+
+>Do users change the forwarding settings for their aliases?
+
+>Do users who have not installed the Private Relay add-on, choose to install the add-on?
+
+>Do users who have not received invitations to the beta choose to join the beta waitlist?
+
+
+## Collection
+
+Events are reported using the [Google Analytics Measurement Protocol](https://developers.google.com/analytics/devguides/collection/protocol/v1/).
```

Comment:
We use [`analytics.js`](https://developers.google.com/analytics/devguides/collection/analyticsjs) on the site, right? Not the measurement protocol?